Kickstarter-iOS/Views/Controllers/ProjectPamphletContentViewController.swift

```diff
@@ -113,7 +113,8 @@ public final class ProjectPamphletContentViewController: UITableViewController {
   internal func commentsViewController(for project: Project) -> UITableViewController {
     let isCommentThreadingFeatureEnabled = AppEnvironment.current.optimizelyClient?
```

Comment:
This isn't being used. Thinking you hard-coded true below for ease of development?

Reply:
Yes, good catch, I'll put in a `//TODO:` For now until this PR is ready to merge. Just leave this comment unresolved.